contracts/PetPark.sol

```diff
@@ -1,7 +1,82 @@
 //SPDX-License-Identifier: Unlicense
 pragma solidity ^0.8.0;
 
-
 contract PetPark {
+    address owner;
+    enum AnimalTypes { None, Fish, Cat, Dog, Rabbit, Parrot}
+    enum Gender { Male, Female }
+
+    struct CallerInfo {
+        uint age;
+        Gender gender;
+        bool returningCaller;
+    }
+
+    mapping(uint => uint) public animalCounts;
+    mapping(address => uint) borrowedAnimalCount;
+    mapping(address => uint) borrowedAnimalType;
+    mapping(address => CallerInfo) callerNameAndAge;
+
+    event Added(uint indexed animalType, uint indexed count);
+    event Borrowed(uint indexed animalType);
+    event Returned(uint indexed animalType);
+
+    constructor() {
+        owner = msg.sender;
+    }
+
+    modifier onlyOwner() {
+        require(owner == msg.sender, "Not an owner");
+
+        _;
+    }
+
+    function add(uint _animalType, uint _count) external onlyOwner {
```

Comment:
You can define this function as `function add(AnimalType _animalType, uint _count)`